templates/federated_reporting/dump.sh

```diff
@@ -76,3 +76,13 @@ else
   mv "$in_progress_file" "$CFE_FR_TRANSPORT_DIR/$CFE_FR_FEEDER.sql.$CFE_FR_COMPRESSOR_EXT"
   chown "$CFE_FR_FEEDER_USERNAME" "$CFE_FR_TRANSPORT_DIR/$CFE_FR_FEEDER.sql.$CFE_FR_COMPRESSOR_EXT"
 fi
+
+if [ -n "CFE_FR_SUPERHUB_HOSTKEYS" ]; then
```

Comment:
Not sure if I really need this if here. The dump.sh file should be linked to the policy which provides the data.

Reply:
e.g. even with only one superhub for this feeder, `$CFE_FR_SUPERHUB_HOSTKEYS` would be defined and so the message might change to

```
log "Copying for superhub(s): $CFE_FR_SUPERHUB_HOSTKEYS"
```

Reply:
+1, I like the suggested change in log message.